examples/MerminMagic/Project.toml

```diff
@@ -0,0 +1,3 @@
+[deps]
+Yao = "5872b779-8223-5990-8dd0-5abbb0748c8c"
+YaoPlots = "32cfe2d9-419e-45f2-8191-2267705d8dbc"
```

Comment:
Please add a [compat] section. Otherwise, the example might break in the future. Please Check other examples to understand how it works.